app/src/main/java/umc/everyones/lck/presentation/lck/AboutLckTeamFragment.kt

```diff
@@ -0,0 +1,82 @@
+package umc.everyones.lck.presentation.lck
+
+import android.os.Bundle
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.content.res.ResourcesCompat
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.fragment.findNavController
+import com.google.android.material.tabs.TabLayout
+import dagger.hilt.android.AndroidEntryPoint
+import umc.everyones.lck.R
+import umc.everyones.lck.databinding.FragmentAboutLckTeamBinding
+import umc.everyones.lck.presentation.base.BaseFragment
+import umc.everyones.lck.presentation.lck.data.PlayerData
+import umc.everyones.lck.presentation.lck.util.OnPlayerItemClickListener
+
+@AndroidEntryPoint
+class AboutLckTeamFragment : BaseFragment<FragmentAboutLckTeamBinding>(R.layout.fragment_about_lck_team),
+    OnPlayerItemClickListener {
+
+    override fun initObserver() {
+
+    }
+
+    override fun initView() {
+        val tabLayout: TabLayout = binding.tbAboutLckTeam
+        val backButton: ImageView = binding.ivAboutLckTeamPre
+        val nextButton: ImageView = binding.ivAboutLckTeamNext
+
+        tabLayout.addOnTabSelectedListener(object : TabLayout.OnTabSelectedListener {
+            override fun onTabSelected(tab: TabLayout.Tab?) {
+                val navHostFragment = childFragmentManager.findFragmentById(R.id.fcv_about_lck_team) as NavHostFragment
+                val navController = navHostFragment.navController
+                when (tab?.position) {
+                    0 -> navController.navigate(R.id.aboutLckRoasterFragment)
+                    1 -> navController.navigate(R.id.aboutLckCoachesFragment)
+                    2 -> navController.navigate(R.id.aboutLckClRoasterFragment)
+                    else -> throw IllegalArgumentException("Invalid tab position")
+                }
+            }
+
+            override fun onTabUnselected(tab: TabLayout.Tab?) {
+            }
+
+            override fun onTabReselected(tab: TabLayout.Tab?) {
+            }
+        })
```

Comment:
```suggestion
        with(binding){
            vp.adapter = VPA(this@AboutLckTeamFragment)

            TabLayoutMediator(tabLayout, vp) { tab, position ->
                tab.text = tabTitles[position]
            }.attach()
        }
```
화면 하단에 navigatio을 하나 더 두는 것보다 
tabLayout이랑 viewPager 연동해서 구현하는 것이 더 좋을거 같아요
수정 부탁드릴게여~~

다만 naming은 제가 임의로 한거라 naming convention 따라서 다시 작성해주세여